pkg/scheduler/profile/profile.go

```diff
@@ -70,6 +70,19 @@ func (m Map) HandlesSchedulerName(name string) bool {
 	return ok
 }
 
+// If the plugins satisfy the io.Closer interface, they are closed.
+// Close calls Close method of each framework.
```

Comment:
This function doesn't need to know about the implementation detail (= plugins implement io.Closer).

```suggestion
// Close closes all frameworks registered in this map.
```